specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/examples/getDiagnosticSettingsCategory.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters" : {
+    "resourceUri" : "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6",
+    "name": "WorkflowRuntime",
+    "api-version" : "2017-05-01-preview"
+  },
+  "responses" : {
+    "200" : {
+      "headers" : {},
+      "body" : {
+        "id":"/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/providers/microsoft.insights/diagnosticSettingsCategories/WorkflowRuntime",
+        "type":"microsoft.insights/diagnosticSettingsCategories",
+        "name":"WorkflowRuntime",
+        "location":"eastus",
+        "tags":null,
+        "properties": {
```

Comment:
`tag` and `location` aren't allowed in `#/definitions/DiagnosticSettingsCategoryResource`